lang_python.adoc

```diff
@@ -33,68 +33,94 @@ Runtime library API version must match compiler version that you're using
 (i.e. if you use ksc v0.8, then you must use compatible v0.8 runtime).
 
 [[add-runtime-release]]
-==== Using release version
+==== Using release version (NOT recommended)
 
 With stable version of Kaitai Struct compiler, one can use use following
 methods:
 
 * On Debian / Ubuntu, one can install it from apt repositories:
 
-** For Python 2.x:
+** For Python 3.x:
 +
 [source,shell]
-sudo apt-get install python-kaitaistruct
+sudo apt-get install python3-kaitaistruct
 +
-Reference: https://packages.debian.org/buster/python-kaitaistruct[package info for Debian], https://packages.ubuntu.com/eoan/python-kaitaistruct[package info for Ubuntu]
+Reference: https://packages.debian.org/buster/python3-kaitaistruct[package info for Debian], https://packages.ubuntu.com/eoan/python3-kaitaistruct[package info for Ubuntu]
 
-** For Python 3.x:
+** For Python 2.x (DEPRECATED, python 2 is EOL since 2020-01-01):
```

Comment:
Let's actually move this Python version debate into separate section above everything? That section could contain a list of officially supported versions + a note about depreciation?